### PR TITLE
Integration kfluxinfra 1143 - part III

### PR DESCRIPTION
**Pull request by meyrevived**
Adding a SeaLIghts scan on the merged main branch, with some code changes for SeaLights testing. 

SeaLights requested we will have a SeaLights scan running on the main branch as well and since I would like to keep SeaLights out of any image building process, I added a SeaLights scan that will be run on the main branch after a pull request has been merged.

### CODE REVIEW

File: .github/workflows/sealights-mpc-scan-on-merged-main.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+name: Run A SeaLights Scan Main Branch After A Pull Request Has Been Merged
+
+on:
+  pull_request:
+    types:
+      - closed
```
[the remark this answers is not in the document]
Comment (meyrevived) [reply .github/workflows/sealights-mpc-scan-on-merged-main.yaml:6]: That's what this is for:
`if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'main'`

File: pkg/aws/ec2.go [hunk 2]
```diff
@@ -239,6 +239,9 @@ func (r AwsDynamicConfig) checkInstanceConnectivity(ctx context.Context, instanc
 }
 
 func pingSSHIp(ctx context.Context, ipAddress string) (string, error) {
+	// THIS CODE WILL BE REMOVED SOON
+	log := logr.FromContextOrDiscard(ctx)
+	log.Info("Just adding some code changes to test SeaLights")
```
Comment (hugares) [inline pkg/aws/ec2.go:244]: let's make sure this does not end up in prod, this will pollute the logs

**Review by hugares — APPROVED** [review]
/lgtm

File: pkg/aws/ec2.go (hunk 2 above)
Comment (meyrevived) [reply pkg/aws/ec2.go:244]: Of course, on it tomorrow morning

**Pull request merged**